### PR TITLE
DOC-4314

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-4314

### CODE REVIEW

**Comment by jamesnocentini** [comment]
@rajagp this is ready for your review.

File: modules/ROOT/pages/why-sync-gateway.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+= Why Sync Gateway?
+
+Couchbase brings the power of NoSQL to mobile.
```
Comment (rajagp) [inline modules/ROOT/pages/why-sync-gateway.adoc]: "Couchbase Mobile" brings the power ...

File: modules/ROOT/pages/why-sync-gateway.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+To do so, it is comprises of three different components: xref:couchbase-lite::index.adoc[Couchbase Lite], an embedded NoSQL database to easily include offline capabilities in your app, Sync Gateway, an internet-facing synchronization mechanism that securely syncs data between device and cloud, and xref:server:introduction:intro.adoc[Couchbase Server], a highly scalable and performant NoSQL database in the cloud.
```
Comment (rajagp) [inline modules/ROOT/pages/why-sync-gateway.adoc]: Replace "To do so, it is comprises of three different components" with "It is comprised of three components"

**Comment by rajagp** [comment]
@jamiltz  For some reason, I cannot make suggestions to commit and its getting bit tedious to make comments.
Can you please look at the intro section of the various samples that I have - Including  https://docs.couchbase.com/userprofile-couchbase-mobile/standalone/userprofile/userprofile_basic.html and https://docs.couchbase.com/userprofile-couchbase-mobile/sync/userprofile/userprofile_sync.html and pick and choose from intro sections . Thanks

File: modules/ROOT/pages/why-sync-gateway.adoc (hunk 1 above)
Comment (jamesnocentini) [reply modules/ROOT/pages/why-sync-gateway.adoc]: Would it be possible to use GitHub's suggestion feature? I think it contributes to a better workflow :) Thanks!

Comment (jamesnocentini) [reply modules/ROOT/pages/why-sync-gateway.adoc]: Oh I just read your later comment.

**Comment by jamesnocentini** [comment]
Sure. i had not seen those.

**Comment by jamesnocentini** [comment]
I'll ping you again when ready for re-review.

**Pull request merged**